lib/modules/home/components/slide_item.dart

```diff
@@ -2,21 +2,49 @@ part of '../home_module.dart';
 
 class SlideItem extends StatelessWidget {
   final SurveyInfo survey;
+  final bool _isEmptied;
+
   const SlideItem({
     Key? key,
     required this.survey,
-  }) : super(key: key);
+  })  : _isEmptied = false,
+        super(key: key);
+
+  const SlideItem._({
+    Key? key,
+    required this.survey,
+    bool isEmptied = false,
+  })  : _isEmptied = isEmptied,
+        super(key: key);
+
+  factory SlideItem.empty({
+    Key? key,
+  }) {
+    final survey = SurveyInfo();
+    survey.title = "";
+    survey.description = "";
+    survey.coverImageUrl = "";
```

Comment:
redundant here, if we do not set `title` or `description`, they will be null as default -> use default value instead of empty string